bin/lolcommits

```diff
@@ -8,72 +8,32 @@ rescue LoadError
   require 'lolcommits'
 end
 
+require 'lolcommits/cli/fatals'
+require 'lolcommits/cli/launcher'
+require 'lolcommits/cli/process_runner'
+require 'lolcommits/cli/history_animator'
+
 include Lolcommits
+include Lolcommits::CLI
 
-require 'launchy'
 require 'choice'
-
 require 'methadone'
 include Methadone::CLILogging
```

Comment:
maybe overkill, but perhaps we should also move the logging into something like `lolcommits/cli/logging` that wraps the  `Methadone::CLILogging` calls?